### ISSUE TITLE
Panel showing boot page only

### ISSUE THREAD

**Issue by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#950]
_Originally posted by @iJuiceman in https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/936#issuecomment-1644800396_

I have the same issue, except I am on 3.4.
The whole flashing process worked fine, but once the GUI comes up on the screen it just hangs.
I have reflashed the NSP with the blank TFT and then the EU one, with the same result.  Looking at the logs, I see:

Logger: homeassistant.components.automation.bed_ns_panel
Source: components/esphome/__init__.py:623
Integration: Automation (documentation, issues)
First occurred: July 20, 2023 at 2:57:25 PM (254 occurrences)
Last logged: 9:46:50 AM

Bed NS Panel: Choose at step 2: choice 29: Error executing script. Unexpected error for call_service at pos 2: Connection not done for bednspanel @ 192.168.107.147; current state is ConnectionState.INITIALIZED!
Bed NS Panel: Choose at step 2: choice 30: Error executing script. Unexpected error for call_service at pos 2: Connection not done for bednspanel @ 192.168.107.147; current state is ConnectionState.INITIALIZED!
Bed NS Panel: Error executing script. Unexpected error for choose at pos 2: Connection not done for bednspanel @ 192.168.107.147; current state is ConnectionState.INITIALIZED!
While executing automation automation.bed_ns_panel
Bed NS Panel: Choose at step 2: choice 19: Error executing script. Unexpected error for call_service at pos 1: Connection not done for bednspanel @ 192.168.107.147; current state is ConnectionState.INITIALIZED!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 468, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 703, in _async_call_service_step
    response_data = await self._async_run_long_action(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 665, in _async_run_long_action
    return long_task.result()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 1965, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2005, in _execute_service
    return await cast(
           ^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/esphome/__init__.py", line 623, in execute_service
    await entry_data.client.execute_service(service, call.data)
  File "/usr/local/lib/python3.11/site-packages/aioesphomeapi/client.py", line 1260, in execute_service
    self._check_authenticated()
  File "/usr/local/lib/python3.11/site-packages/aioesphomeapi/client.py", line 328, in _check_authenticated
    self._check_connected()
  File "/usr/local/lib/python3.11/site-packages/aioesphomeapi/client.py", line 322, in _check_connected
    raise APIConnectionError(
aioesphomeapi.core.APIConnectionError: Connection not done for bednspanel @ 192.168.107.147; current state is ConnectionState.INITIALIZED!

_Originally posted by @iJuiceman in https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/936#issuecomment-1644800396_

**Comment by edwardtfn** [comment]
> aioesphomeapi.core.APIConnectionError: Connection not done for bednspanel @ 192.168.107.147; current state is ConnectionState.INITIALIZED!

Looks like your panel is not properly connected to Home Assistant.
Can you see the device page under Settings > Devices & Services > ESPHome?

**Comment by iJuiceman** [comment]
Yep, and it was detected. I can fully control it from the d&s page. I can also trigger the tft update.
When I power on the NSP I see the IP pop on the screen after about 2 seconds. 
I tried flashing the eu tft and it did the same thing as the US. I have also tried the blank, which worked as well.
I am going to pull it apart and make sure the ribbon cable has not dislodged as it has to be some weird thing like that
